sites/partners/tailwind.config.js

```diff
@@ -8,4 +8,15 @@ const bloomTheme = cloneDeep(require("@bloom-housing/ui-components/tailwind.conf
 // For example:
 // bloomTheme.theme.colors.white = "#f0f0e9"
 
-module.exports = bloomTheme
+module.exports = {
+  ...bloomTheme,
+  purge: {
+    enabled: true, // in future we can remove the explicit enabling and purge only in production
```

Comment:
why not do the purging only in prod now?

Reply:
We could, just not sure how confident we are that this is working perfectly. If it purges in development we can see immediately if something is off when working on the codebase. But if folks think we should change this, fine with me.￼